packages/plugins/RedPacket/src/SiteAdaptor/hooks/useAvailabilityComputed.ts

```diff
@@ -22,16 +25,24 @@ export function useAvailabilityComputed(account: string, payload: RedPacketJSONP
         chainId: parsedChainId,
     })
 
-    const { data: password = payload.password } = useQuery({
+    const author = usePostInfoDetails.author()
+    const platform = usePlatformType()
+    const signedMsg = useSignedMessage({
+        account,
+        contractVersion: payload.contract_version,
+        password: payload.password,
+        rpid: payload.rpid,
+        profile: platform ? { platform, profileId: author?.userId || '' } : undefined,
+    })
+    const { data: password = signedMsg } = useQuery({
```

Comment:
This looks strange to me. A password and a signature are not the same thing.

Reply:
Actually, it's the `signedMsg` parameter to the contract.